run.py

```diff
@@ -3,22 +3,16 @@
 from random import random
 from random import randint
 
-buyers = []
-sellers = []
-auctioneer = Auctioneer(buyers, sellers)
+
+auctioneer = Auctioneer()
 
 for _ in range(Config.run_length):  # run model!
     if random() < Config.random_agent_creation:
-        _ = randint(1,4)
-        # We should assume that demand is higher than supply and that there are more sellers than buyers.
-        # However, since distribution of Q for both buyers and sellers is equal, this results in a lower
-        # demand than supply (needs to be changed in config/buyer/seller?). Therefore, I changed it a
-        # bit so that there are more buyers than sellers, but supply < demand.
-        if _ < 4:
+        _ = randint(1, 4)
```

Comment:
Use an explicit name if you are going to use the variable. An underscore is generally used in places where the variable is not actually used, like the above loop (also, this underscore `hides' the outer-scope one, which is generally considered bad practice).